src/index.ts

```diff
@@ -181,59 +200,50 @@ client.on('voiceStateUpdate', async (oldState, newState) => {
   }
 });
 
-enum PresenceState {
-  IN_CHAMP_SELECT = 'In Champion Select',
-  IN_GAME = 'In Game',
-  IN_LOBBY = 'In Lobby',
-  IN_QUEUE = 'In Queue'
-}
-
 // Event triggered when a user's presence (e.g. status, activity, etc.) is changed.
 client.on('presenceUpdate', async (_, newPresence) => {
-  // Make sure this update is for someone present in the voice channel
+  // Make sure this update is for someone present in the same VoiceChannel as the bot
   const member = newPresence.member;
-  if (member && channel && member.voice.channelId == channel.id) {
-    // Determine if there are any new activities to report
-    const newActivity = newPresence.activities[0];
-    if (newActivity) {
-      // Determine if this presence update is because a user is now playing a game
-      if (newActivity.type === ActivityType.Playing) {
-        // Make sure the game is League of Legends
-        if (newActivity.name === 'League of Legends') {
-          // Determine the state of the activity
-          console.log(newActivity.state);
-          switch (newActivity.state) {
-            case PresenceState.IN_LOBBY:
-            case PresenceState.IN_QUEUE:
-            case PresenceState.IN_CHAMP_SELECT: {
-              break;
-            }
-            case PresenceState.IN_GAME: {
-              if (leagueOfLegendsPollIntervalId === null && IS_LOL_ANNOUNCER_ENABLED) {
-                console.log('Polling game...');
-                leagueOfLegendsPollIntervalId = pollCurrentGame(
-                  channel,
-                  audioPlayer,
-                  PATH_TO_CLIPS
-                );
-              }
-              break;
-            }
-            default: {
-              if (leagueOfLegendsPollIntervalId !== null) {
-                clearInterval(leagueOfLegendsPollIntervalId);
-                console.log('No longer in game, polling stopped...');
-                leagueOfLegendsPollIntervalId = null;
-              }
-              break;
-            }
-          }
+  if (
+    member &&
+    channel &&
+    member.voice.channelId == channel.id &&
```

Comment:
Double equals! Can we enforce that via ESLint, prettier or tsconfig?

Reply:
Whoops! Yeah looks like there's a rule you can implement (https://eslint.org/docs/latest/rules/eqeqeq), I'll add it

Reply:
Also good with moving the lint rule to a separate ticket

Reply:
I went ahead and added it as a new commit to this branch https://github.com/bryangottschalk/dudes-discord-bot/pull/32/commits/c5deaf620ec29e5dd9984f0a835ea0268bb33511